feature_annotations.yaml

```diff
@@ -0,0 +1,28 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  documented_elsewhere:
+    - ".. documented_elsewhere_name:":
+    - ".. documented_elsewhere:":
+  feature_toggle:
+    - ".. feature_toggle_name:":
+    - ".. feature_toggle_type:":
+        choices: [switch, rollout, group, ConfigurationModel, feature_flag]
```

Comment:
What is the exact semantic for this `feature_toggle_type` field?  If it's intended to be how it is implemented, then we should rename it and have choices such as: `[configuration_model, waffle_flag, django_setting]`.

Reply:
I think @robrap's [comment](https://github.com/edx/edx-platform/pull/19715/files#r252737849) makes sense of this.

My original intent was to follow the toggle report described in [OEP-0017](https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#req-12-report), and at the same time include information that would be useful to me as an operator -- whether the toggle was a feature_flag, a Waffle or a ConfigurationModel. It would be clearer to separate the type and the implementation.